### PR TITLE
PyHEP WG page - add PyHEP 2022 workshop info

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Getting things ready for a community-wide announcement of the workshop very soon :-).

### CODE REVIEW

File: _workinggroups/pyhep.md [hunk 1]
```diff
@@ -45,15 +45,38 @@ The **PyHEP workshops** are a series of workshops initiated by Eduardo Rodrigues
...
-| ---------- | ---------------------- | ---------------------| ----------------------------------------------------------------------------- |
-| PyHEP 2021 | Virtual workshop       | July 5-9, 2021       | [Indico](https://indico.cern.ch/e/PyHEP2021){:target="_pyhep2021_indico"}     |
-| PyHEP 2020 | Virtual workshop       | July 13-17, 2020     | [Indico](https://indico.cern.ch/e/PyHEP2020){:target="_pyhep2020_indico"}     |
-| PyHEP 2019 | Abingdon, U.K.         | October 16-18, 2019  | [Indico](https://indico.cern.ch/e/PyHEP2019){:target="_pyhep2019_indico"}     |
-| PyHEP 2018 | Sofia, Bulgaria        | July 7-8, 2018       | [Indico](https://indico.cern.ch/event/694818/){:target="_pyhep2018_indico"}   |
+| *Workshop* | *Location*             | *Date*                | *Agenda link*                                                                 |
+| ---------- | ---------------------- | ----------------------| ----------------------------------------------------------------------------- |
+| PyHEP 2021 | Virtual workshop       | September 12-16, 2022 | [Indico](https://indico.cern.ch/e/PyHEP2022){:target="_pyhep2022_indico"}     |
```
Comment (graeme-a-stewart) [inline _workinggroups/pyhep.md:50]: Should be `PyHEP 2022`

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Just a tiny typo to fix 2021 -> 2021
We should also pop this onto the front page of the site, when we are ready (AE2 is now old news!)

File: _workinggroups/pyhep.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply _workinggroups/pyhep.md]: Oh, and I did check ... I thought! Thanks, fixed!

**Comment by eduardo-rodrigues** [comment]
> We should also pop this onto the front page of the site, when we are ready (AE2 is now old news!)

See it. Good point. Added to the to-do list - have to check where that gets defined. This MR was more important because Indico refers to the page.

Thanks a lot.

**Pull request merged**